tests/Auth0.ManagementApi.IntegrationTests/OrganizationTests.cs

```diff
@@ -152,26 +152,31 @@ public async Task Test_organization_connections_crud_sequence()
             var createConnectionResponse = await fixture.ApiClient.Organizations.CreateConnectionAsync(ExistingOrganizationId, new OrganizationConnectionCreateRequest
             {
                 ConnectionId = ExistingConnectionId,
-                AssignMembershipOnLogin = true
+                AssignMembershipOnLogin = true,
+                ShowAsButton = true
```

Comment:
`show_as_button` defaults to true on the API, can we remove this line to ensure the default value is used when omitted?

Reply:
I've removed that line and also removed the equivalent in the update call so there is a test for backwards compatability, as well as making `ShowAsButton` nullable on `OrganizationConnectionUpdateRequest` and `OrganizationConnectionCreateRequest`. This means `Test_organization_connections_crud_sequence` no longer tests creates and updates with `ShowAsButton` set, I've added `Test_organization_connections_show_as_button_crud_sequence` as an attempt to test this, or would you prefer this be tested by extending `Test_organization_connections_crud_sequence`?

Reply:
See https://github.com/auth0/auth0.net/pull/706#discussion_r1530113404, I think we can stick to that.